frontend/src/views/Profile.vue

```diff
@@ -25,7 +25,7 @@
         <api-date-picker
           :name="$tc('entity.user.fields.birthday')"
           :uri="profile._meta.self"
-          :value-format="$moment.HTML5_FMT.DATE"
+          :value-format="'YYYY-MM-DD'"
```

Comment:
A constant for the date format would be nice.
I would add it when we need this literal another time.
Do we need the additional quotes (" **'** **'** ")

Reply:
Thank you